go.mod

```diff
@@ -1,15 +1,20 @@
 module github.com/extism/go-sdk
 
-go 1.20
+go 1.22
 
 require (
+	github.com/dylibso/observe-sdk/go v0.0.0-20240815143955-7e0389165219
```

Comment:
Be sure to bump this after merging https://github.com/dylibso/observe-sdk/pull/172